README.md

```diff
@@ -102,23 +144,30 @@ The emulator can also be stopped by closing the command window.
 
 - The configuration of the overall emulator is controlled by the *emulator-config.json* file in the directory where the emulator is installed. (This is the **Swordfish** folder in the installation steps above.) Instructions for using this file can be found in the Redfish Interface Emulator *README.md* file.
 
-- The Swordfish API Emulator code files generally do not overlap the Redfish Interface Emulator code files, but there are currently two files that must contain code for both the Redfish Interface Emulator and the Swordfish API Emulator:
-  - *api_emulator\resource_manager.py*
-  - *api_emulator\utils.py*
+- The Swordfish API Emulator leverages the general structure of the Redfish Interface Emulator and some of the utility functions, but replaces most of the code for the Redfish and Swordfish services and objects.  This service functions in a fully dynamic mode, meaning that objects generally support modification according to the Redfish and Swordfish specifications.
+
+- The Swordfish API Emulator supports the following headers:
+  --  RESP_HEADERS_CONTENT_TYPE
+  --  REQ_HEADERS_ODATA_VERSION
+  --  RESP_HEADERS_ALLOW_GET_OR_HEAD
+  --  RESP_HEADERS_CACHE_CONTROL
+  --  RESP_HEADERS_LINK_REL_DESCRIBED_BY
+  --  RESP_HEADERS_ALLOW_METHOD_NOT_ALLOWED
+
+- The dynamic resources in the emulator can be populated via the emulator API using Create/Read/Update/Delete (CRUD) operations.  The Swordfish emulator supports all instances of Redfish and Swordfish objects, via all URI paths.  If additional objects are desired, the AutoGen tool can be used to create additional objects, and the paths can be added to  `resource_manager.py()` in the api_emulator\redfish directory.
 
-- The *api_emulator\resource_manager.py* file establishes which emulator resources are static and which emulator resources are dynamic. Create/Read/Update/Delete (CRUD) operations can be done on dynamic resources via the emulator API using REST operations, but static resources are read-only and cannot be changed via the emulator API.
+- The following services are implemented as read-only services: <br>
```

Comment:
If you add indentation here and use a single `-` you would avoid needing the `<br>` breaks.